### PR TITLE
Change Hyperdrive's upper serial number offset.

### PR DESCRIPTION
**Pull request by Hydreigon223**
This now makes the game read its PIC as the correct ID instead of a development PIC. Verified this from a vague picture of a spare Hyperdrive pic showing the game ID.

Other changes:
- Remove player 3 and 4 inputs for games that don't use them

### CODE REVIEW

File: src/mame/drivers/seattle.cpp [hunk 1]
```diff
@@ -2277,7 +2286,7 @@ void seattle_state::hyprdriv(machine_config &config)
 
 	MIDWAY_IOASIC(config, m_ioasic, 0);
 	m_ioasic->set_shuffle(MIDWAY_IOASIC_HYPRDRIV);
-	m_ioasic->set_upper(469); //unknown
+	m_ioasic->set_upper(471); // others?
```
Comment (angelosa) [inline src/mame/drivers/seattle.cpp:2289]: `// unconfirmed value`

File: src/mame/drivers/seattle.cpp [hunk 2]
```diff
@@ -1400,7 +1400,10 @@ static INPUT_PORTS_START( seattle_analog )
 
 	PORT_MODIFY("SYSTEM")
 	PORT_BIT( 0x0004, IP_ACTIVE_LOW, IPT_START1 ) PORT_NAME("Start Button")
-	PORT_BIT( 0x0020, IP_ACTIVE_LOW, IPT_UNUSED )
+	PORT_BIT( 0x0620, IP_ACTIVE_LOW, IPT_UNUSED )
+
+	PORT_MODIFY("IN2")
+	PORT_BIT( 0xffff, IP_ACTIVE_LOW, IPT_UNUSED  )
```
Comment (angelosa) [inline src/mame/drivers/seattle.cpp:1406]: Are these really `IPT_UNUSED` or just `IPT_UNKNOWN`? Have you checked if games doesn't read these locations in any way or form, even in test mode?

Comment (Hydreigon223) [reply src/mame/drivers/seattle.cpp:1406]: Didn't really know that ipt unused and unknown had a slight but major difference so going to change these accordingly. Other than that, the seattle_analog games all consist of racing games which I noticed IN2 (normally controls players 3 and 4 for the joystick games) are left unconnected for these games.

File: src/mame/drivers/seattle.cpp (hunk 1 above)
Comment (Hydreigon223) [reply src/mame/drivers/seattle.cpp]: Would "no alternates" work?

**Pull request merged**